app/src/main/java/com/nytimes/android/sample/activity/PersistingStoreActivity.java

```diff
@@ -62,8 +63,9 @@ public void loadPosts() {
                 .toList()
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(this::showPosts, throwable -> {
-                });
+                .subscribe(this::showPosts, throwable ->
+                        Log.e(StoreActivity.class.getSimpleName(), throwable.getMessage(),
+                                throwable));
```

Comment:
It is consistent with https://github.com/NYTimes/Store/blob/feature/rx2/app/src/main/java/com/nytimes/android/sample/activity/StoreActivity.java#L73